tlsfuzzer/messages.py

```diff
@@ -566,10 +566,19 @@ def _generate_extensions(self, state):
             if ext_id == ExtensionType.renegotiation_info:
                 ext = RenegotiationInfoExtension()\
                     .create(state.key['client_verify_data'])
-                extensions.append(ext)
+            elif ext_id == ExtensionType.status_request:
+                ext = StatusRequestExtension().create()
+            elif ext_id in (ExtensionType.client_hello_padding,
+                            ExtensionType.encrypt_then_mac,
+                            ExtensionType.extended_master_secret,
+                            35,  # session_ticket
```

Comment:
wouldn't it be better to have named variables here instead of number + comments ?

Reply:
it would, but they are defined in tlslite-ng (and not really supported), so it's a stop-gap measure